R/markdown.R

```diff
@@ -183,7 +183,10 @@ mdxml_children_to_rd <- function(xml, state) {
 mdxml_node_to_rd <- function(xml, state) {
   if (!inherits(xml, "xml_node") ||
       ! xml_type(xml) %in% c("text", "element")) {
-    roxy_tag_warning(state$tag, "Internal markdown translation failure")
+    warn_roxy_tag(state$tag, c(
+      "markdown translation failed",
+      "internal error"
```

Comment:
Not related to this PR, but I wonder if we could be more helpful here. Even if we just say "please open an issue"?